<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowUi</class>
 <widget class="QWidget" name="MainWindowUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BCT Controller</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Scelta dispositivi</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>MIDI Input</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="tuioCheckBox">
          <property name="text">
           <string>Tuio</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>40</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Porta</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="tuioPortSpinBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximum">
             <number>65336</number>
            </property>
            <property name="value">
             <number>3333</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Scelta Modello BCT</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinAndMaxSize</enum>
        </property>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinAndMaxSize</enum>
          </property>
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::FieldsStayAtSizeHint</enum>
          </property>
          <property name="rowWrapPolicy">
           <enum>QFormLayout::DontWrapRows</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>IP</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="ipAddress">
            <property name="text">
             <string>131.175.18.122</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Porta</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="bctPort">
            <property name="maximum">
             <number>65336</number>
            </property>
            <property name="value">
             <number>6984</number>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="loadModels">
            <property name="text">
             <string>Scarica Modelli</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="startBct">
       <property name="text">
        <string>Avvia Modelli selezionati</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>tuioCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>tuioPortSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
